### ISSUE TITLE
Missing tooltip + button too big on Thunderbird

### ISSUE THREAD

**Issue by sphh** [fastaddons/ScrollAnywhere#32]
**Describe the bug**
Missing tooltip + button too big on Thunderbird before and after #31:
![Screenshot from 2021-03-03 09-45-01](https://user-images.githubusercontent.com/4856462/109779667-11793d80-7bfe-11eb-9324-ad249cb44bb3.png)

Firefox has a small button and a working tooltip (before upgrading to 8.7):
![Screenshot from 2021-03-03 09-45-35](https://user-images.githubusercontent.com/4856462/109779335-c3fcd080-7bfd-11eb-8556-b2e07d8900c4.png)

**Steps To Reproduce**
1. Move you mouse over the toolbar in Thunderbird.

**Expected behavior**
I would expect for a tooltip with some contents to show.

I would also expect that the button is not larger than needed.

**Device info:**
 - operating system: Linux Mint 20.1
 - Browser and version: Thunderbird 78.7.1 (64-bit), [Firefox 86.0 (64-bit)]
 - addon version: 8.6 and 8.7 (Thunderbird), [8.6 (Firefox)]

**Comment by fastaddons** [comment]
Yeah, that's a compromise :)
Your Thunderbird is set to show "Icons beside Text" (when you right click / Customize, you can see the option in the bottom).
So it should actually show the icon with a text "ScrollAnywhere 8.7". Not exactly a short thing to have in a toolbar, especially considering you'll almost never have to use it.

So I've decided to remove the text and keep icon only - but that doesn't work so I had to keep a single space there " ". And that's the reason there is no tooltip - because if I set one, it will be shown as text next to icon.
And that's the story of how the crappy toolbar icon came to be :D.

The only good solution I see it to make a feature request for Thunderbird to allow addons to add toolbar icons without text.
I will do that later today.

**Comment by sphh** [comment]
Ah, I see. Thanks for explaining.

Just a stupid idea: Could you use a HAIR SPACE (` `, " ", U+200A) or even a ZERO WIDTH SPACE (`​`, "​", U+200B) instead of the SPACE (` `, " ", U+0020)? This does not resolve the underlying problem, but might improve the button a bit.

**Comment by fastaddons** [comment]
There is a zero width space??? :D, Cool! I've learned something new today! Thanks!
I'm totally gonna try that.

**Comment by sphh** [comment]
I only knew, that there are spaces with different width. So I was looking for the thinnest available space and came across the zero width space … Hope it works!

**Comment by fastaddons** [comment]
Oh man! Almost!!!
It would work but the parent node for the icon has min-width set:
```css
toolbar[mode="full"] .toolbarbutton-1:not([hideWebExtensionLabel="true"]) {
    min-width: 55px;
}
```
You can see it when you inspect the toolbar with Developer Tools "Ctrl + Shift + i".

But this is helpful, all I need to do is ask them to remove it :).

**Comment by fastaddons** [comment]
Reported here:
https://bugzilla.mozilla.org/show_bug.cgi?id=1696147
Let's see what they say about it.

**Comment by sphh** [comment]
I could the button to shrink with the following in `chrome/userChrome.css`:
```css
#juraj_masiar_gmail_com_scrollanywhere-browserAction-toolbarbutton {
        /*min-width: 0 !important;*/
        min-width: unset !important;
}
```
Either of the two `min-width` line work, but the second might be might be the cleaner option (and the first line might want to have `0 px` instead of `0`).

**Comment by fastaddons** [comment]
Yeah, that's one way to fix it! :)
But still, it would be nice to have a proper fix that would work for everyone by default.

**Comment by sphh** [comment]
I discovered purely by accident, that you can put icons near the ‘Switch to the calendar tab’, the ‘Switch to the tasks tab’ and the ‘Switch to the CardBook tab’ (https://github.com/Rayquaza01/changelogger/issues/21#issuecomment-1019162561). This also works for the `ScrollAnywhere` icon! In that place, no space is displayed beside the icon.
It's sad, that the Thunderbird developers haven't replied to your bug report…

**Comment by fastaddons** [comment]
Yeah, they are a very small team so I can imagine they have no time for "cosmetic" bugs.
I'm in a similar situation right now - I've build many addons and each requires some of my time and now I have so little time for each of them.
